ambry-messageformat/src/main/java/com.github.ambry.messageformat/MessageFormatRecord.java

```diff
@@ -282,6 +297,18 @@ public static BlobAll deserializeBlobAll(InputStream stream, StoreKeyFactory sto
      */
     short getVersion();
 
+    /**
+     * @return whether this message header has an life version.  Life Version counts how many times
+     * a blob has been undeleted
+     */
+    boolean hasLifeVersion();
```

Comment:
Remove it?

Reply:
Yeah since it's likely that we'll just be using '0' as the default... I'll keep it for now but if I find it's not necessary I'll dump it

Reply:
ok.